themes-default/slim/views/partials/showheader.mako

```diff
@@ -10,64 +10,47 @@
 <%namespace file="/inc_defs.mako" import="renderQualityPill"/>
 
 <div class="row">
-    <div id="showtitle" class="col-lg-12" data-showname="${show.name | h}">
+    <div id="showtitle" class="col-lg-12" :data-showname="show.title">
```

Comment:
Can't find any usage for `data-showname` (or `data('showname')`) in the code.
Can probably be removed.

Reply:
Shall I skip this for now? I could add a `@TODO` comment.

Reply:
Sure, go ahead with a `@TODO`.